helm/examples/azure/hpcc-azurefile/templates/_helpers.tpl

```diff
@@ -61,3 +61,82 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Add common mountOptions
+*/}}
+{{- define "hpcc-azurefile.addCommonMountOptions" }}
+mountOptions:
+- uid=10000 # uid of user 'hpcc'
+- gid=10001 # gid of group 'hpcc'
+- dir_mode=0700 # user read/write/execute
+- file_mode=0600 # user read/write
+{{- if .root.Values.common.shareName }}
+- mfsymlinks
+- nobrl
+{{- end -}}
+{{- end }}
+
+{{/*
+Create PersistentVolume
+*/}}
+{{- define "hpcc-azurefile.addStoragePV" -}}
+{{- $plane := .plane -}}
+{{- $common := .root.Values.common -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ printf "%s-%s-pv" $plane.name (include "hpcc-azurefile.fullname" .root) }}
+  labels:
+    store: {{ printf "%s-%s-pv" $plane.name (include "hpcc-azurefile.fullname" .root) }}
+spec:
+  capacity:
+    storage: {{ $plane.size }}
+  accessModes:
+    - {{ $plane.rwmany | default false | ternary "ReadWriteMany" "ReadWriteOnce" }}
+  azureFile:
+    secretName: {{ $common.secretName }}
+    secretNamespace: {{ $common.secretNamespace }}
+    shareName: {{ $plane.shareName }}
+    readOnly: false
+  {{- include "hpcc-azurefile.addCommonMountOptions" . | indent 2}}
+{{- end -}}
+
+{{/*
+Create StorageClass name
+*/}}
+{{- define "hpcc-azurefile.SCName" -}}
+{{- $plane := .plane -}}
```

Comment:
trivial: do you need $plane, could you not just use .plane directly below?

Reply:
Yes, removed all $plane and just use .plane instead. Checked with "helm lint"